documentation/source/product-model.rst

```diff
@@ -683,6 +700,56 @@ The CDM supports Product Qualification functions for Credit Derivatives, Interes
 
 The output of the qualification function is used to populate the ``productQualifier`` attribute of the ``ProductTaxonomy`` object, which is created when a ``ContractualProduct`` object is created. The product taxonomy includes both the product qualification generated by the CDM and any additional product taxonomy information which may come from the originating document, such as FpML. In this case, taxonomy schemes may be associated to such product taxonomy information, which are also propagated in the ``ProductTaxonomy`` object.
 
+<<<<<<< HEAD
```

Comment:
This HEAD section seems to appear in the rich diff - will this also appear when the file is displayed? If so it duplicates the next paragraph and there is also an "(icma-documentation-update)" tag